.env.example

```diff
@@ -1,10 +1,21 @@
+VERSION=0.0.1
+VERSION_TAG=latest
+
 DATABASE_ADDRESS=localhost
 DATABASE_USER=postgres
 DATABASE_PASSWORD=postgres
 DATABASE_PORT=5432
 DATABASE_NAME=postgres
 
+WORKERS_AMOUNT=5
+DRIVER=docker
+WORKER_ADDRESS=tcp://localhost:5555
+
+# if ALLOW_ALL is false, an allowed_list file must be provided so it should be path passed in ALLOWED_LIST_PATH
```

Comment:
"an allow_list file must be provided" sounds enough

Reply:
addressed